enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/InteractiveQuestion.unit.spec.tsx

```diff
@@ -98,6 +107,37 @@ describe("InteractiveQuestion", () => {
     ).toBeInTheDocument();
   });
 
+  it("should render loading state when drilling down", async () => {
+    const { store } = setup();
+
+    await waitForLoaderToBeRemoved();
+
+    expect(
+      within(screen.getByTestId("TableInteractive-root")).getByText(
+        TEST_COLUMN.display_name,
+      ),
+    ).toBeInTheDocument();
+    expect(
+      within(screen.getByRole("gridcell")).getByText("Test Row"),
+    ).toBeInTheDocument();
+
+    expect(screen.queryByTestId("loading-spinner")).not.toBeInTheDocument();
+    // Mimicking drilling down by rerunning the query again
+    const storeDispatch = store.dispatch as unknown as ThunkDispatch<
+      State,
+      void,
+      AnyAction
+    >;
+    storeDispatch(clearQueryResult());
+    storeDispatch(runQuestionQuery());
```

Comment:
These 2 actions do:
1. clear the results so it's as if we're about to query a new question when we're drilling
1. query again